src/modules/gfx/packet.cpp

```diff
@@ -85,6 +85,13 @@ void gfxPacket::GetTri(int* out, int index)
     out[2] = Indices[(3 * index) + 2];
 }
 
+void gfxPacket::SetTri(int* tri, int index)
```

Comment:
Could you make this function signature
(int i0, int i1, int i2, int index) instead?

Reply:
Signature improved!